### DISCUSSION TITLE
Issues with upgrading from 3.1.1 to 3.2.0

### DISCUSSION THREAD

**Discussion by wmarucha** [netbox-community/netbox#9255] [Q&A]
Hello all,

I have been trying to upgrade our netbox server from version 3.1.1 to 3.2.0 for the past few days but have been running an issue that I have not seen mentioned anywhere else. Using option A from the official documentation, I am able to get the upgrade script to complete without any errors, but when accessing the web interface I see that I can no longer change the amount of results displayed per page. I am also seeing an error with fetching options for custom fields when accessing the change log. For further context, we recently upgraded from 2.11.12 > 3.1.0 and then to 3.1.1 which appears to have been operating just fine. After encountering the issue upgrading from 3.1.1 to 3.2.0, I restored a snapshot of the server from before and attempted to upgrade to 3.1.2 but still ran into the issue of being unable to change the amount of results displayed per page. I am still rather new to netbox so it is very possible that I am missing something small.

**Comment by wmarucha** [comment]
Update on further testing I have done. Started with a fresh db and installed 3.2 using our existing ldap_config.py, configuration.py, gunicorn.py and local_requirements.txt files. Even with a fresh db, after creating a batch of test entries I am still unable to change the number of entries displayed per page and am unable to search vm names. That would lead me to assume that there is something in the configuration file that would be causing these issues, though running diff on it and the example config included in the 3.2 install did not reveal anything. Any and all ideas are appreciated as I am running out of things to try. 

![netbox-search](https://user-images.githubusercontent.com/104593337/166700876-aeb37ec3-8b9b-4a25-8d1b-987ea0f25421.JPG)
[local_requirements.txt](https://github.com/netbox-community/netbox/files/8621107/local_requirements.txt)
[gunicorn.txt](https://github.com/netbox-community/netbox/files/8621112/gunicorn.txt)
[ldap_config.txt](https://github.com/netbox-community/netbox/files/8621113/ldap_config.txt)
[configuration.txt](https://github.com/netbox-community/netbox/files/8621114/configuration.txt)

**Reply by markkuleinio** [reply]
> am unable to search vm names

Based on the screenshot I assume you mean the live filter entry that says "test". I'm not a NetBox GUI expert but I think that field relies on javascript and other browser-based things, so I'd recommend clearing the caches from the browser, and/or trying with another browser.

**Reply by wmarucha** [reply]
Tried with edge, chrome and firefox. Same issues on each one unfortunately.
